### PR TITLE
[ISSUE 104] DB Planning milestone doc

### PR DESCRIPTION
**Pull request by widal001**
### Summary

Creates the milestone doc for DB Planning

Resolves: #104 

### Changes Proposed

- Creates `documentation/milestones/individual_milestones/db_planning.md`
- Adds `DB-Plan` node to milestone dependency diagram
- Updates dependencies in milestone short description doc

### Instructions for Review

- Review `documentation/milestones/individual_milestones/db_planning.md`
- Confirm that updated dependencies match expectations

### Additional Notes

There is a follow-up ticket to answer the remaining open questions raised by @gcarson-bit #247

### CODE REVIEW

**Review by acouch — APPROVED** [review]

File: documentation/milestones/individual_milestones/db_planning.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+- Is this deployment option FedRAMP compliant?
+- How commonly used is this deployment option? Is it easy for new developers to learn?
+- How expensive is it to host a database with this option, both in terms of direct operating costs and in terms of team resources?
+- Does the service offer features like automatic backups or read replicas?
+
+### Definition of done
+<!-- Required -->
+
+- [ ] The tools and services chosen have been authorized under the Grants.gov ATO (Authority to Operate)
```
Comment (lucasmbrown-usds) [inline documentation/milestones/individual_milestones/db_planning.md:101]: Same edit as from the previous PR: based on Julius's comment, let's change this to: have been authorized under the Grants.gov ATO (Authority to Operate) and/or are FedRAMP approved.

File: documentation/milestones/individual_milestones/db_planning.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+Are there any notable capabilities / milestones do NOT we expect to be in place at the
+beginning of work on this milestone?
+
+- [x] None
+
+### Open questions
+<!-- Optional -->
+
+- [ ] Will this database serve as an OLTP system, OLAP system, or both? Will this database be used for analytics, and is there a need for a second database to be created with data pipelines?
+- [ ] Is the existing Grants.gov database heavy on read or write transactions?
```
Comment (lucasmbrown-usds) [inline documentation/milestones/individual_milestones/db_planning.md:136]: Note there are thee main existing Grants.gov databases: 

* Oracle on RDS (used for opportunity data, etc)
* MongoDB (I'm not sure exactly what data is in the document store)
* Flat file storage with application data

**Review by lucasmbrown-usds — APPROVED** [review]
LGTM!

File: documentation/milestones/individual_milestones/db_planning.md (hunk 2 above)
Comment (widal001) [reply documentation/milestones/individual_milestones/db_planning.md:140]: Thanks! Added these notes and created a follow-up ticket (#247) for the other questions.

File: documentation/milestones/individual_milestones/db_planning.md (hunk 1 above)
Comment (widal001) [reply documentation/milestones/individual_milestones/db_planning.md]: Added this note in [this commit](https://github.com/HHS/grants-equity/pull/115/commits/6a42496e92d9ba5f78eb5cd3cb90c829de4295a2)

**Pull request merged**